desk/app/groups-ui.hoon

```diff
@@ -238,4 +254,28 @@
   ?+  wire  !!
     [%build ~]  cor
   ==
+++  preload-contacts
+  =+  .^(chat-running=? (scry %gu %chat /$))
+  =?  cor  chat-running
+    =+  .^  [dms=(map ship dm:c) *]
+        (scry %gx %chat /full/noun)
+      ==
```

Comment:
Usually you would align `==` with the opening rune, but perhaps here it is ok? @Fang-